dagmc_stats/DagmcQuery.py

```diff
@@ -99,5 +138,62 @@ def calc_tris_per_vert(self, ignore_zero=True):
         if self._vert_data.empty:
             self._vert_data = self._vert_data.append(t_p_v_data)
         else:
-            self._vert_data.set_index('vert_eh').join(
-                self._vert_data.append(t_p_v_data).set_index('vert_eh'))
+            self._vert_data.set_index('vert_eh').join(self._vert_data.append(t_p_v_data).set_index('vert_eh'))
+
+    def calc_triangle_aspect_ratio(self):
+        """
+        Calculate triangle aspect ratio data (according to the equation:
+        (abc)/(8(s-a)(s-b)(s-c)), where s = .5(a+b+c).)
+
+        inputs
+        ------
+        none
+
+        outputs
+        -------
+        none
+        """
+        t_a_r_data = []
+        tris = self.get_tris()
+        for tri in tris:
+            side_lengths = list(self.get_tri_side_length(tri).values())
```

Comment:
is it redundant to do `list( )` on this? I think `.values()` on a dictionary already returns a list.